lib/raxx/router.ex

```diff
@@ -67,16 +60,12 @@ defmodule Raxx.Router do
       end
 
     quote location: :keep do
-      @impl Raxx.Server
-      def handle_request(_request, _state) do
-        raise "This callback should never be called in a on #{__MODULE__}."
-      end
-
       @impl Raxx.Server
       unquote(routes)
 
       @impl Raxx.Server
       def handle_data(data, {controller, state}) do
+        # TODO add handle_data etc functions from my middleware branch
```

Comment:
I don't follow, but you probably do :)

Reply:
Yeah I started on general clean up but realised that would confuse this PR